src/slash-bedrock/share/brl-fetch/distros/manjaro

```diff
@@ -0,0 +1,133 @@
+	LC_ALL=C chroot "${1}" /usr/bin/pacman-key --refresh-keys
+
+	if ! grep -q "^Server" "${1}/etc/pacman.d/mirrorlist"; then
+		echo "### Set by Bedrock Linux when acquiring this stratum" >>"${1}/etc/pacman.d/mirrorlist"
+		echo "Server = ${target_mirror}/stable\$repo/\$arch" >>"$1/etc/pacman.d/mirrorlist"
+	fi
+}
+
+fetch() {
+	bootstrap_deps="manjaro-tools-base pacman pacman-mirrors ca-certificates ca-certificates-utils"
+
+	step "Downloading package information database"
+	wget -O "${bootstrap_dir}/core.db.tar.gz" "${target_mirror}/stable/core/${distro_arch}/core.db.tar.gz"
+	wget -O "${bootstrap_dir}/extra.db.tar.gz" "${target_mirror}/stable/extra/${distro_arch}/extra.db.tar.gz"
+	wget -O "${bootstrap_dir}/community.db.tar.gz" "${target_mirror}/stable/community/${distro_arch}/community.db.tar.gz"
+
+	step "Decompressing package information database"
+	mkdir -p "${bootstrap_dir}/pacmandb/core" "${bootstrap_dir}/pacmandb/extra" "${bootstrap_dir}/pacmandb/community"
+	(
+		tar -xv -f "${bootstrap_dir}/core.db.tar.gz" -C "${bootstrap_dir}/pacmandb/core/"
+		tar -xv -f "${bootstrap_dir}/extra.db.tar.gz" -C "${bootstrap_dir}/pacmandb/extra/"
+		tar -xv -f "${bootstrap_dir}/community.db.tar.gz" -C "${bootstrap_dir}/pacmandb/community/"
+	) | awk 'NR%100==0' | progress_unknown
+
+	step "Converting distro package information database to brl format"
+	pacmandb_to_brldb "${bootstrap_dir}/pacmandb" "${bootstrap_dir}/brldb"
+
+	step "Calculating required bootstrap packages"
+	brldb_calculate_required_packages "${bootstrap_dir}/brldb" "${bootstrap_dir}/required_packages" "${bootstrap_deps}"
+
+	step "Downloading bootstrap packages"
+	# brldb contains repo/filename
+	# files are at mirror/stable/repo/arch/filename
+	checksum_downloads "${cache}/packages/" "$(awk -v"a=${distro_arch}" -v"m=${target_mirror}" '{sub("/", "/"a"/"); print m"/stable/"$0}' "${bootstrap_dir}/required_packages")"
+	step "Extracting bootstrap packages"
+	bootstrap_packages="$(awk -v"d=${cache}/packages/" '{sub(/^.*\//,d);print $1}' "${bootstrap_dir}/required_packages")"
+	extract_pacman_pkgs "${bootstrap_dir}" ${bootstrap_packages}
+
+	step "Running bootstrap software"
+	# pacstrap mounts devtmpfs (which is shared with the host system) and
+	# chowns files in it based on /etc/passwd and /etc/group.  Ensure
+	# system copies of files are used to avoid problematic file ownership
+	# change in /dev.
+	mkdir -p "${target_dir}/etc"
+	cp -a "/etc/passwd" "${target_dir}/etc/passwd"
+	cp -a "/etc/group" "${target_dir}/etc/group"
+
+	setup_chroot "${bootstrap_dir}"
+	share_cache "packages" "${bootstrap_dir}/target-root/var/cache/pacman/pkg/"
+	# -a to use the api, -P all protocols, -t 1 timeout if it takes longer than 1 second
+	LC_ALL=C chroot "${bootstrap_dir}" pacman-mirrors -a -P all -t 1
+	# Because there is no default mirrorlist in Manjaro, we have to copy it
+	mkdir -p "${target_dir}/etc/pacman.d"
+	cp "/etc/pacman.d/mirrorlist" "${target_dir}/etc/pacman.d"
```

Comment:
Yes, I meant to copy the mirrorlist from the output of pacman-mirrors. This is a copy / paste fail from my dev script into the brl fetch script. I didn't get an error on the final test before making this PR because I was in an Arch based strata.

As for not using ```pacman-mirrors``` -- no, there probably isn't a good reason to use it. I was trying to stick as close to possible to the actual install routine, and generating a mirrorlist with pacman-mirrors is part of the install. I will go ahead and fix this so that the mirror from ```"${target_mirror}" ```is used.